website/addons/base/views.py

```diff
@@ -40,15 +40,52 @@
 # import so that associated listener is instantiated and gets emails
 from website.notifications.events.files import FileEvent  # noqa
 
-FILE_GONE_ERROR_MESSAGE = u'''
+# TODO - make distiction between filenodes that had GUIDs and were trashed
+# through the OSF/WB and those that were trashed due to not being found at the
+# provider. Currently both cases go through 'FILE_GONE_HAD_GUID'.
+ERROR_MESSAGES = {'FILE_GONE': u'''
 <style>
 .file-download{{display: none;}}
 .file-share{{display: none;}}
 .file-delete{{display: none;}}
 </style>
 <div class="alert alert-info" role="alert">
-This link to the file "{file_name}" is no longer valid.
-</div>'''
+<p>
+This {provider} link to the file "{file_name}" was deleted from the OSF.
+</p>
+<p>
+
+It was delete by <a href="/{deleted_by}">{deleted_by}</a> on {deleted_on}
```

Comment:
1.) s/delete/deleted/   2.) Needs a period at the end.  3.) Can you format `deleted_on`?  Drop the millisecond component and display the timezone (UTC is fine for now).